Config.php

```diff
@@ -106,8 +117,10 @@ public function __construct(\PDO $dbh, $config_source = NULL, $config_type = '',
                 // check xml-config is readable
                 if (!is_readable($source)) die("PHPAuth: config type is FILE, declared as {$source}, but file not readable or not exist"); //@todo: \Exception
 
-                // load configuration
-                $this->config = simplexml_load_string(file_get_content($source));
+                if (!XMLReader::open($source)->isValid()) die("PHPAuth: config type XML is corrupted"); //@todo: \Exception
```

Comment:
This doesn't work by me. I always get false back from `isValid()`. I found nothing in the documentation, but on Stackoverflow (https://stackoverflow.com/a/4554256/2310353)

This code works fine:
```
libxml_use_internal_errors(true);

$xmlstr = simplexml_load_file("test.xml");
if (!$xmlstr) {
  die("PHPAuth: config type XML is corrupted");
  // $errors = libxml_get_errors(); // <- returns xml erros
}
```

**So I think the following code is right:**

```
libxml_use_internal_errors(true); // don't display errors
$config = simplexml_load_file($source);

if(!$config){
   die("PHPAuth: config type XML is corrupted");
}

$this-config = $config;
```